jetty9/src/main/docker/root.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+<Configure id="webapp" class="org.eclipse.jetty.webapp.WebAppContext">
+  <Set name="contextPath">/root</Set>
+  <Set name="war"><Property name="jetty.webapps" default="."/>/root.war</Set>
+  <Set name="extractWAR">true</Set>
+  <Set name="tempDirectory"><Property name="jetty.base" default="."/>/webapps/root-exploded</Set>
```

Comment:
So I checked the exact logic on the jetty unpacker and to unpack from webapps/root.war to webapps/root/   you need to set extractWAR to true (done) AND the timestamp of the war file needs to be more recent than the timestamp of the directory.  So I think you can probably achieve what you want with:
````
mkdir webapps/root
touch --date 1970/01/01 webapps/test
````

Or better yet, just arrange for the root.war to be unpacked by a Dockerfile, so it happens as the image is built rather than when the image is run.